multi/stm32l4-multi/stm32l4-multi.c

```diff
@@ -240,14 +243,22 @@ int main(void)
 
 	priority(THREADS_PRIORITY);
 
-#if !CONSOLE_IS_TTY
-	/* called before init functions to redirect logs here */
+#if CONSOLE_IS_TTY
+	portCreate(&common.ttyConsolePort);
+#endif
 	portCreate(&common.port);
+
+#if BUILTIN_DUMMYFS
+	fs_init();
```

Comment:
#else wait for rootfs to register?

Reply:
I thinking if it's valid. When you want to have rootfs on some real memory, then you can need multi to operate with it, but multi will not start until rootfs is not established… 

But as it's theoretical, I change it as you said.